content/en/blog/general/apply-vap-using-cli/index.md

```diff
@@ -0,0 +1,222 @@
+---
+date: 2023-09-04
+title: "Applying Validating Admission Policies using Kyverno CLI"
+linkTitle: "Applying Validating Admission Policies using Kyverno CLI"
+author: Mariam Fahmy
+description: "Using Kyverno CLI to apply Validating Admission Policies"
+---
+The [Kyverno Command Line Interface (CLI)](https://kyverno.io/docs/kyverno-cli/) is designed to validate and test policy behavior to resources prior to adding them to a cluster.
+
+One of the Kyverno CLI commands are apply and test commands.
+
+1. The apply command is used to perform a dry run on one or more policies for the given manifest(s).
+2. The test command is used to test a given set of resources against one or more policies to check the desired results.
+
+Kyverno CLI now can be used to apply/test Kubernetes Validating Admission Policies that were first introduced in 1.26.
+
+## Applying Validating Admission Policies using Kyverno apply
+In this section, you will create a validating admission policy that checks the number of deployment replicas. You will then apply this policy to two deployments, one of which violates the policy:
+
+```sh
+cat << EOF > check-deployment-replicas.yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: check-deployments-replicas
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["apps"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["deployments"]
+  validations:
+    - expression: "object.spec.replicas <= 2"
+      message: "Replicas must be less than or equal 2"
+EOF
+```
+
+The following deployment satisfies the rules declared in the above policy:
+```sh
+cat << EOF > deployment-pass.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-pass
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: nginx-pass
+  template:
+    metadata:
+      labels:
+        app: nginx-pass
+    spec:
+      containers:
+      - name: nginx-server
+        image: nginx
+EOF
+```
+
+Let's apply the policy on the resource using kyverno apply as follows:
+```sh
+kyverno apply ./check-deployment-replicas.yaml --resource deployment-pass.yaml
+
+Applying 1 policy rule(s) to 1 resource(s)...
+
+pass: 1, fail: 0, warn: 0, error: 0, skip: 0 
+```
```

Comment:
Probably best to break this out into two different code blocks: the first just has the `kyverno apply` command while the second has the output. This is since the manifests you've provided above were written such that users should copy-and-paste into a terminal to `cat`-out the file.